app/models/unlink_attempt.rb

```diff
@@ -8,6 +8,10 @@ class UnlinkAttempt
   validates :username, presence: true
   validates :reason_code, presence: true, inclusion: { in: :valid_reason_codes }
   validates :other_reason_text, presence: { if: :text_required? }
+  validates :other_reason_text, length: {
+    maximum: 500,
+    message: I18n.t('activemodel.errors.models.unlink_attempt.attributes.other_reason_text.max_character')
```

Comment:
is it not possible to pick this message translation automatically

Reply:
i think if you just use `.length` and have a message for `length:` you can dispense with the translation here

Reply:
How would I accomplish this?